app/services/EntityService.scala

```diff
@@ -81,9 +82,7 @@ class EntityService @Inject()(val s4LService: S4LService,
 }
 
 object EntityService {
-
   case class ScottishPartnershipName(answer: String)
-
   case class Telephone(answer: String)
-
+  case class EntityEmail(answer: String)
```

Comment:
trivial one, but can we name this `Email` as the context is already abt entity and will be inline with other case classes. thanks